app/controllers/application_controller.rb

```diff
@@ -17,6 +17,11 @@ def health_check
     render plain: health_check_haiku, status: 200
   end
 
+  def robots
+    robots = File.read(Rails.root + "config/robots.#{Settings.robots}.txt")
+    render plain: robots
+  end
```

Comment:
I am now realizing there are other actions that have found their way into application controller and I am v sad. If you found you had time to move them all to home controller I would be just chuffed.